packages/wdio-config/src/utils.ts

```diff
@@ -150,3 +149,8 @@ export function loadBabelCompiler (babelOpts: Record<string, any> = {}, requireS
         return false
     }
 }
+export function ensureAbsolutePathForSpecs(specFileList: string[]) {
+    return specFileList.map((specFile) => {
+        return new FileSystemPathService().ensureAbsolutePath(specFile)
```

Comment:
Let's instantiate the service once at the top of the file

Reply:
Fixed in the latest commit